besu/src/main/java/org/hyperledger/besu/cli/util/CommandLineUtils.java

```diff
@@ -76,8 +76,8 @@ public static void checkOptionDependencies(
    * @param isMainOptionCondition the conditions to test dependent options against. If all
    *     conditions are true, dependent options will be checked.
    * @param dependentOptionsNames a list of option names that can't be used if condition is met.
-   *     Example: if --min-gas-price is in the list and condition is that either --miner-enabled or
-   *     --goquorum-compatibility-enabled should not be false, we log a warning.
+   *     Example: if --min-gas-price is in the list and condition is that --miner-enabled should not
```

Comment:
prob should still have an example that has multiple option names even if you have to make it up. Or maybe we don't need this method?